### DISCUSSION TITLE
please add the commands by reations

### DISCUSSION THREAD

**Discussion by anxlz** [jagrosh/MusicBot#602] [Ideas]
please add the commands by reations and make it to play from spotify playlist

**Comment by jagrosh** [comment]
As per the page of [things that won't be added](https://github.com/jagrosh/MusicBot/wiki/Things-That-Won%27t-Be-Added#spotify), Spotify support will not be added.

As for commands by reactions, that seems very generic; maybe it would be worth discussing what that would entail in more detail.
